Doc/tools/check-epub.py

```diff
@@ -1,24 +1,30 @@
-import sys
 from pathlib import Path
 
+CPYTHON_ROOT = Path(
+    __file__,  # cpython/Doc/tools/check-epub.py
+    '..',  # cpython/Doc/tools
+    '..',  # cpython/Doc
+    '..',  # cpython
+).resolve()
```

Comment:
```suggestion
CPYTHON_ROOT = Path(__file__).parents[2]  # cpython/Doc/tools/check-epub.py -> cpython/
```

Reply:
The current change is more intuitive with the comment.